src/main/org/deidentifier/arx/metric/v2/ILScore.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * ARX: Powerful Data Anonymization
+ * Copyright 2012 - 2018 Fabian Prasser and contributors
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.deidentifier.arx.metric.v2;
+
+import org.apache.commons.math3.fraction.BigFraction;
+import org.deidentifier.arx.metric.InformationLoss;
+
+/**
+ * This class implements a score value in the form of an information loss,
+ * with appropriate comparison semantics (i.e. higher score values are better).
+ * 
+ * @author Raffael Bild
+ *
+ */
+public class ILScore extends InformationLoss<BigFraction> {
+
+    /** SVUID. */
+    private static final long serialVersionUID = -2638719458508437194L;
+
+    /** Value */
+    private BigFraction                 value;
+
+    /**
+     * Creates a new instance.
+     *
+     * @param value
+     */
+    ILScore(final BigFraction value) {
+        this.value = value;
+    }
+    
+    @Override
+    public InformationLoss<BigFraction> clone() {
+        return new ILScore(getValue());
+    }
+
+    @Override
+    public int compareTo(InformationLoss<?> other) {
+        if (!other.getClass().equals(this.getClass())) {
+            throw new IllegalArgumentException("Incompatible class (" +
+                    other.getClass().getSimpleName() +
+                    ")");
+        }
+        BigFraction otherValue = ((ILScore)other).getValue();
+        return value.compareTo(otherValue) * -1;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) return true;
+        if (obj == null) return false;
+        if (getClass() != obj.getClass()) return false;

```

Comment:
Is this really needed? Maybe this should be removed for efficiency reasons (but I don't know how often this method is actually used...)